### PR TITLE
[bugfix] git hash fix position to bottom

### PR DESCRIPTION
**Pull request by AlejandroRigau**
*Issue #, if available:* N/A

*Description of changes:*
- git hash fix position to bottom

To reproduce the bug, make the screeen small, then scroll the side navbar.

Before:
Screenshot 2024-04-16 at 9 57 22 AM

After:
Screenshot 2024-04-16 at 9 58 06 AM

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by dustins — APPROVED** [review]

**Comment by AlejandroRigau** [comment]
Seems that this didnt fix it completely. Ill push another commit for it. thanks @KyleLilly for pointing it out

Screenshot 2024-04-16 at 11 10 29 AM

**Review by douglas1850 — APPROVED** [review]

**Comment by AlejandroRigau** [comment]
Now it sticks to the bottom using flexbox and does not overlap the nav components

Screenshot 2024-04-17 at 12 09 35 PM

File: frontend/src/shared/layout/navigation/side-navigation.tsx [hunk 1]
```diff
@@ -178,18 +183,19 @@ export default function SideNavigation () {
...
-                            fontSize: '10px',
-                            color: '#969696'
-                        }}
-                        title={window.gitInfo?.gitHash}>{window.gitInfo?.revisionTag}</div>
-                </Box>
+                <div
+                    style={{
+                        fontSize: '10px',
+                        marginTop: 'auto',
+                        color: '#969696',
```
Comment (dustins) [inline frontend/src/shared/layout/navigation/side-navigation.tsx:190]: I know we have theme-able styling now. Just want to confirm this color should be hard coded as opposed to something more dynamic. I think @LightSeekerSC did the them stuff so perhaps he can weigh in?

Comment (AlejandroRigau) [reply frontend/src/shared/layout/navigation/side-navigation.tsx:190]: This color was already there but I can change it to anything that is more appropriate. Not related to my current changes though.

**Review by dustins — DISMISSED** [review]

File: frontend/src/shared/layout/navigation/side-navigation.tsx (hunk 1 above)
Comment (KyleLilly) [reply frontend/src/shared/layout/navigation/side-navigation.tsx:190]: From a theming perspective we should be able to just ditch the color all together - it's going to default to the correct color based on the theme.

**Comment by AlejandroRigau** [comment]
This is ready for review

**Review by KyleLilly — APPROVED** [review]

**Review by dustins — APPROVED** [review]

**Pull request merged**